backend/run/copr-repo

```diff
@@ -358,9 +358,6 @@ def main_locked(opts, batch, log):
     # possible we should move those two things before 'delete_builds' call.
     add_appdata(opts)
 
-    # while we still hold the lock, notify others we processed their task
-    batch.commit()
```

Comment:
Race 2 closed here; we don't do double-commit -- the second one possibly bringing the task ID back into the loop for no reason.